src/courtroom.cpp

```diff
@@ -344,6 +346,9 @@ Courtroom::Courtroom(AOApplication *p_ao_app) : QMainWindow()
   connect(ui_take_that, SIGNAL(clicked()), this, SLOT(on_take_that_clicked()));
   connect(ui_custom_objection, SIGNAL(clicked()), this,
           SLOT(on_custom_objection_clicked()));
+  connect(ui_custom_objection,
+          SIGNAL(customContextMenuRequested(const QPoint &)), this,
+          SLOT(ShowContextMenu(const QPoint &)));
```

Comment:
This signal seems to be busted... Maybe look into how 2.8 QMenus are being called and use that instead? (Such as the music list rightclick context menu)

Reply:
```suggestion
          SLOT(show_custom_objection_menu(const QPoint &)));
```
This is why I don't like old-style signals and slots - they don't catch nonexistent or incompatible functions at compile time.